hack/pre-release.sh

```diff
@@ -0,0 +1,221 @@
+
+    SUBSCRIPTION="openshiftGitOps"
+    CHANNEL="latest"
+    SOURCE="gitops-iib"
+}
+
+configure_pipelines(){
+    # PIPELINES_IMAGE="quay.io/openshift-pipeline/openshift-pipelines-pipelines-operator-bundle-container-index"
+    # PIPELINES_IMAGE_TAG="v4.17-candidate"
+
+    SUBSCRIPTION="openshiftPipelines"
+    CHANNEL="latest"
+    SOURCE="pipelines-iib"
+}
+
+configure_rhdh(){
+    RHDH_INSTALL_SCRIPT="https://raw.githubusercontent.com/redhat-developer/rhdh-operator/main/.rhdh/scripts/install-rhdh-catalog-source.sh"
+    curl -sSLO $RHDH_INSTALL_SCRIPT
+    chmod +x install-rhdh-catalog-source.sh
+
+    ./install-rhdh-catalog-source.sh --latest --install-operator rhdh
+
+    SUBSCRIPTION="redHatDeveloperHub"
+    CHANNEL="fast-1.5"
+    SOURCE="rhdh-fast"
+}
+
+configure_rhtas() {
+    # Configure CatalogSource
+    echo '
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: rhtas-operator
+  namespace: openshift-marketplace
+spec:
+  sourceType: grpc
+  grpcPodConfig:
+    securityContextConfig: legacy
+  publisher: Red Hat
+  image: quay.io/securesign/fbc-v4-18@sha256:124d3fcff0c711ba8d03a405a36b0e128a900065e0687885e626e0b4153ec804
+  displayName: TAS Operator
+' \
+    | oc apply -f -
+
+    # Configure ImageDigestMirrorSet
+    echo '
+apiVersion: config.openshift.io/v1
+kind: ImageDigestMirrorSet
+metadata:
+  name: registry-stage
+spec:
+  imageDigestMirrors:
+    - mirrors:
+        - registry.stage.redhat.io/rhtas
+      source: registry.redhat.io/rhtas
+' \
+    | oc apply -f -
+
+    # Configure pull-secret
+    DOCKERCONFIGJSON=$(
+        oc get secrets \
+            -n openshift-config pull-secret \
+            -o jsonpath='{.data.\.dockerconfigjson}' \
+        | base64 -d
+    )
+
+    DOCKERCONFIGJSON=$(
+        echo "$DOCKERCONFIGJSON" \
+        | jq -c '.auths."registry.stage.redhat.io" = {
+            "auth": "MTEwMDkxMDN8cmhhZHM6ZXlKaGJHY2lPaUpTVXpVeE1pSjkuZXlKemRXSWlPaUk0T0RsaE1XWTJPR0ZsWVdVMFptTTNZVE13WWpBNVlXUmpOR1JsWVRneVlpSjkuSXZrVDZqQ001ZU9wUnJQeDVzOVk4NFlaY1NfcWpDX0RpZ0s1bmwtd3NCQ0Y4cGFVdG13b25RbE5GTm9sYXBQTGZUeFFUaWxUUDluYllIRGV4TGxGWnZUWGNRWmg4YllqMmFGOUhZaHR2bFpCbmxpS0tyakNVN24tUE5rb09DY296QmdfN21fZFU4Z1lIUGt4Q3pISC1obG53Y2dneklwMG5jV0xQTDhFTkM4R0lJV2pQZ0t4bm1vTDQ2TUNUUDl6amhNNG5LYUlnT1RuNzdCZGVqRHY4WGRoZVNKRHFJUEEyTjA0TnUwMWhaSE95U3NpX3NCdjR2OThZYllOdFhNLUtHSXdtYWtMcWNZTzFONFUwa0RJaUc5OWdMZ2V0ZGVYT1ZPWkl3ajVpRGFkZXd2cnoxdFlucGZvZ1JhT2pWU3pDcnVBTDkyM1IzdEV2M3FvMjNPWXdXS1hGZlBTMU9YbzBwNFE3YldKRUFmMnEtM1gxM0lOQ19UTmJ3MGZCRHdMS05XN1k2TEVoQmJZVFNZd1BET20tTTJSVXpMQzdKYjV1c2pOZjYxM01XZWt5RmVRVVlJSTNUc0xLeUExQ0s1cE9SdXNFS256MERjZ1lya0V0ZE9ENkVtYjBmdUdNbElhN0JiX1UwZjVxUmszdFFXbWdTejRkQ1VDMXhlM0NDT25kVGU4bDA2RUVzSGVVU2I5NUd0bWtudUVkME03eERZaURjUURRVXlQblhmRFl5WF9xcF9BZ2l1M19meExLcl9LOGplYzJlcHlIUFBFN3FhR2hHeEViVVlUc0dsZ1B0Z0ExUFRlS3h1dklyM0d6ajg4SF85bGVYU1lwR0d5Ujl6dXh1TXVNaFZFS19Jc25OYTl3bHBaTG1RdW0xM1lRelN5ajMtMFVWUEVma3M=",
```

Comment:
Is this supposed to be a secret looks like a jwt token?

Reply:
I'm surprised by the format as well, but that's what was shared by the TAS team, and it was working fine in my tests.